Code/Tools/ProjectManager/Source/CreateAGemScreen.cpp

```diff
@@ -0,0 +1,446 @@
+
+        m_gemSystemName = new FormLineEditWidget(tr("Gem System name*"), "");
+        QLabel* gemSystemNameErrorLabel = m_gemSystemName->getErrorLabel();
+        gemSystemNameErrorLabel->setObjectName("createAGemFormErrorLabel");
+        QLineEdit* gemSystemLineEdit = m_gemSystemName->lineEdit();
+        gemSystemLineEdit->setPlaceholderText(tr("The name used in the O3DE Gem System"));
+        secondScreen->addWidget(m_gemSystemName);
+
+        FormLineEditWidget* gemSummary = new FormLineEditWidget(tr("Gem Summary"), "");
+        QLineEdit* gemSummaryLineEdit = gemSummary->lineEdit();
+        gemSummaryLineEdit->setPlaceholderText(tr("A short description of your Gem"));
+        secondScreen->addWidget(gemSummary);
+
+        FormLineEditWidget* requirements = new FormLineEditWidget(tr("Requirements"), "");
+        QLineEdit* requirementsLineEdit = requirements->lineEdit();
+        requirementsLineEdit->setPlaceholderText(tr("Notice of any requirements your Gem. i.e. This requires X other gem"));
+        secondScreen->addWidget(requirements);
+
+        m_license = new FormLineEditWidget(tr("License*"), "");
+        QLabel* licenseErrorLabel = m_license->getErrorLabel();
+        licenseErrorLabel->setObjectName("createAGemFormErrorLabel");
+        QLineEdit* licenseLineEdit = m_license->lineEdit();
+        licenseLineEdit->setPlaceholderText(tr("License uses goes here: i.e. Apache-2.0 or MIT"));
+        secondScreen->addWidget(m_license);
+
+        FormLineEditWidget* licenseURL = new FormLineEditWidget(tr("License URL"), "");
+        QLineEdit* licenseURLLineEdit = licenseURL->lineEdit();
+        licenseURLLineEdit->setPlaceholderText(tr("Link to the license web site i.e. https://opensource.org/licenses/Apache-2.0"));
+        secondScreen->addWidget(licenseURL);
+
+        FormLineEditWidget* origin = new FormLineEditWidget(tr("Origin"), "");
+        QLineEdit* originLineEdit = origin->lineEdit();
+        originLineEdit->setPlaceholderText(tr("The name of the originator goes. i.e. XYZ Inc."));
+        secondScreen->addWidget(origin);
+
+        FormLineEditWidget* originURL = new FormLineEditWidget(tr("Origin URL"), "");
+        QLineEdit* originURLLineEdit = originURL->lineEdit();
+        originURLLineEdit->setPlaceholderText(tr("The primary repo for your Gem. i.e. http://www.mydomain.com"));
+        secondScreen->addWidget(originURL);
+
+        FormLineEditWidget* userDefinedGemTags = new FormLineEditWidget(tr("User-defined Gem Tags"), "");
+        secondScreen->addWidget(userDefinedGemTags);
+
+        FormFolderBrowseEditWidget* gemLocation = new FormFolderBrowseEditWidget(tr("Gem Location"), "");
+        QLineEdit* gemLocationLineEdit = gemLocation->lineEdit();
+        const char* gemLocationPlaceholderText = R"(C:\O3DE\Project\Gems\YourGem)";
+        gemLocationLineEdit->setPlaceholderText(tr(gemLocationPlaceholderText));
+        secondScreen->addWidget(gemLocation);
```

Comment:
This should really be doing something similar to what [NewProjectSettingsScreen does where it sets the default location based on a default name and project path](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/NewProjectSettingsScreen.cpp#L49).

Reply:
The placeholder text was decided by UX, leaving it in for now.